styles.css

```diff
@@ -114,3 +117,346 @@
+}
+
+.flex-container ul {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  font-weight: 600;
+  font-size: 13px;
+  height: 16px;
+  width: 236px;
+  padding-left: 16px;
+}
+
+.blue {
+  color: #344563;
+}
+
+.canopy {
+  width: 295px;
+  height: 76px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 12px;
+  margin-bottom: 5px;
+  padding-left: 16px;
+}
+
+.tags ul {
+  display: flex;
+  list-style: none;
+  padding: 4px 12px;
+  gap: 30px;
+  letter-spacing: 0.03em;
+  margin-top: 12px;
+}
+
+.tag1 {
+  width: 53px;
+  height: 24px;
+  border-radius: 8px;
+  background: #ebebff;
+}
+
+.tag2 {
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.tag3 {
+  width: 89px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-bottom: 1px 108px 100px 130px;
+}
+
+.nav-button {
+  height: 48px;
+  width: 124px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  box-sizing: border-box;
+  position: absolute;
+  color: #396df2;
+  padding: 12px;
+  gap: 10px;
+  margin-left: 12px;
+  margin-bottom: 28px;
+}
+
+.mybutton {
+  width: 100px;
+  height: 24px;
+  border: none;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  background: none;
+  color: #396df2;
+}
+
+/* Section 3 */
+
+.border_section3 {
+  width: 327px;
+  height: 596px;
+  box-sizing: border-box;
+  margin: 24px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  margin-bottom: 88px;
+}
+
+.image_section3 {
+  width: 295px;
+  height: 220px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  border-radius: 8px;
+  padding: 16px;
+  background: url(image_processing20200714-9747-16mjqpg.png);
+}
+
+.border_heading {
+  width: 295px;
+  height: 124px;
+  margin-top: 12px;
+}
+
+.heading {
+  width: 295px;
+  height: 80px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding-left: 16px;
+  margin-bottom: 10px;
+  margin-top: 12px;
+}
+
+/* Section 4 */
+
+.border_section4 {
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border-radius: 16px;
+  margin: 24px;
+  box-sizing: border-box;
+  border: 1px solid #dfe1e6;
+  margin-bottom: 88px;
+}
+
+/* Section 5 */
+
+.border_section5 {
+  width: 327px;
+  height: 596px;
+  box-sizing: border-box;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+/* Section 6 */
+
+.heading_resume {
+  width: 327;
+  height: 101px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-top: 1px;
+  margin-bottom: 6px;
+}
+
+.container_resume {
+  width: 375px;
+  height: 1263px;
+  background: #fff;
+  margin-top: 114px;
+}
```

Comment:
- [ ] It is required that the main container of about myself section has a `display: flex` for the layout flexibility. Kindly apply `display: flex` to the main container of about myself section.